### ISSUE TITLE
About multi-model ensembling

### ISSUE THREAD

**Issue by ytrhhiaya** [dvlab-research/SphereFormer#59]
I saw that you mentioned that multi-model ensembling can help to  improve the accuracy of the test set. How much mIOU  can multi-model ensembling improve on the test set? 
 By the way, How to adjust the hyper-parameters such as lr, weight decay ？I adjusted the learning rate to 0.01 and the weight delay to 0.01, but the performance actually decreased. Looking forward to your answer very much. Thank you！
